server/protocols/jmap/src/main/java/org/apache/james/jmap/methods/GetMessagesMethod.java

```diff
@@ -39,7 +39,9 @@
 import org.apache.james.mailbox.MailboxSession;
```

Comment:
WIP GetMessages looks like a terrible commit message... I wonder who did it... Could you fix it ?